crt_portal/cts_forms/templates/forms/complaint_view/show/attachments.html

```diff
@@ -14,11 +14,19 @@
     <input type="hidden" value="{{ return_url_args }}" name="next" id="next" />
     <input type="hidden" value="{{ index }}" name="index" id="index" />
     <input type="hidden" value="{{ data.pk }}" name="report" id="report_id" />
-    {% for attachment in attachments %}
-      <p>{{ attachment.filename }}</p>
-    {% endfor %}
+        <table class="usa-table usa-table--borderless related-reports width-full">
```

Comment:
We should include headers describing the content of each cell when using tables to display data. Headers let folks know what to expect in a given cell and allow screen readers to provide context about the data as folks navigate the table.

Reply:
Great catch!! we talked about this earlier for table accessibility (508).  Sorry forgot to add that, should be in our next push. thanks.